shared/teams/team/member/index.new.tsx

```diff
@@ -167,7 +235,19 @@ const SubteamInRow = (props: SubteamInRowProps) => {
     />
   )
 
-  // TODO(Y2K-1291): get this data
+  const [role, setRole] = React.useState<Types.TeamRoleType>(props.membership.type)
```

Comment:
can't have state in a component that's going to be part of a SectionList because the state will go away when it goes offscreen, so you need to move this state up into the parent.

Reply:
counterpoint: it doesn't matter, this is only for purposes of the role picker

if that's not ok, what would be your recommended solution?

Reply:
Ah, I see - you're dispatching as soon as it's selected. It's fine with me.